src/lib/model.ts

```diff
@@ -413,6 +422,15 @@ export class Course {
 		}
 		return false
 	}
+
+	getModule(moduleId: string) {
+		const mods = this.modules.filter(m => m.id === moduleId)
```

Comment:
If possible, use the `find()` function here. It returns one item or returns `undefined` if nothing is found. Saves you from having to return an array and getting the first item.